MovingBlocks/Terasology
# Second sun fix

Fixes the second "setting" sun in the skysphere, that is always present (even during night).

How to test:
Run the game, change the time to the game and ensure you only see one sun during day and no sun during night.
